master/internal/api_workspace.go

```diff
@@ -521,6 +554,117 @@ func (a *apiServer) PatchWorkspace(
 		errors.Wrapf(err, "error refetching updated workspace (%d) from db", currWorkspace.Id)
 }
 
+func (a *apiServer) addWorkspaceNamespaceBinding(ctx context.Context,
+	req *apiv1.AddWorkspaceNamespaceBindingRequest, tx *bun.Tx, wkspID int,
```

Comment:
Why do we need wkspID when AddWorkspaceNamespaceBindingRequest has workspace ID in it

Reply:
Good catch, removed that param! I think I'd initially had the API handler take in the workspace name rather than its ID, and the ID was retrieved separately which is why I  added that param in the first place, but I totally forgot to remove it after I changed the API handler to accept the ID in the request 🙈😂. Either way, just removed it!